src/index.ts

```diff
@@ -349,7 +349,7 @@ task("starknet-deploy", "Deploys Starknet contracts which have been compiled.")
                     const execResult = processExecuted(executed);
                     
                     if(execResult == 0)
-                        tx_hashes.push(executed.stdout.toString().match(".*Transaction hash: (?<tx_hash>\\w*).*").groups["tx_hash"]);
+                    txHashes.push(executed.stdout.toString().match(".*Transaction hash: (?<tx_hash>\\w*).*").groups["tx_hash"]);
```

Comment:
I guess you didn't want this line to look the way it does. Perhaps you meant to indent it and enclose with curly braces. Also, would it be possible to utilize `extractTxHash` from types.ts?